ui/shared/tabs/failureSummary/FailureSummaryTab.jsx

```diff
@@ -96,27 +89,6 @@ class FailureSummaryTab extends React.Component {
           !suggestion.bugs.too_many_open_recent;
       });
 
-      // if we have no bug suggestions, populate with the raw errors from
-      // the log (we can do this asynchronously, it should normally be
-      // fast)
-      if (!suggestions.length) {
-        const { data, failureStatus } = await getData(
-          getProjectJobUrl(textLogErrorsEndpoint, selectedJob.id),
-        );
-        if (!failureStatus && data.length) {
-          const errors = data.map((error) => ({
-            line: error.line,
-            line_number: error.line_number,
-            logViewerUrl: getLogViewerUrl(
-              selectedJob.id,
-              repoName,
-              error.line_number,
-            ),
-          }));
-          this.setState({ errors });
-        }
-      }
```

Comment:
if we remove this block and there are no suggestions, will we not get the error lines printed out?

Reply:
The failure lines in the main view are always retrieved from the `bug_suggestions` endpoint. The code was dead as far as I could recognize and check. The inspection showed `bug_suggestions` unexpectedly also contains the line number, will check tomorrow if we can get rid of the whole `text_log_errors` endpoint (but `bug_suggestions` might be slower than the slimmed down `text_log_errors`).

Reply:
I am not familiar with falureSummaryTab vs bug_suggestions;  I need to somehow determine where in the UI these things live and translate that to names in the code.

Reply:
I spent some time looking over this, I think this is dead code, we can always revert if there are issues